graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -325,7 +325,9 @@ def _repartition_parquet_files_in_memory(
             for idx in range(len(desired_counts))
         ]
         with Parallel(
-            n_jobs=min(NUM_WRITE_THREADS, os.cpu_count()), verbose=self.verbosity, prefer="threads"
+            n_jobs=min(NUM_WRITE_THREADS, os.cpu_count() or 16),
```

Comment:
How do we determine we need to use 16 here?

Reply:
It's a safe number to use according to my testing. In some systems `os.cpu_count()` might return `None` leading to an error here. We provide 16 as a fallback value.